src/modules/comment.js

```diff
@@ -0,0 +1,44 @@
+import commentsCounter from './commentCounter.js';
+
+const baseURL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/';
```

Comment:
✅ The project should use the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) to record the different user interactions (likes, comments and reservations). 👍🏻